src/components/Reports/Shared/CalculationReports/SubmitModal/SubmitModal.tsx

```diff
@@ -12,22 +12,28 @@ import { DateTime } from 'luxon';
 import { useTranslation } from 'react-i18next';
 import { useLocale } from 'src/hooks/useLocale';
 import { dateFormatShort } from 'src/lib/intlFormat';
-import { useMinisterHousingAllowance } from '../Shared/Context/MinisterHousingAllowanceContext';
-import { mocks } from '../Shared/mockData';
+import { useMinisterHousingAllowance } from '../../../MinisterHousingAllowance/Shared/Context/MinisterHousingAllowanceContext';
+import { mocks } from '../../../MinisterHousingAllowance/Shared/mockData';
 import { PageEnum } from '../Shared/sharedTypes';
 
 interface SubmitModalProps {
+  formTitle: string;
   handleClose: () => void;
   handleConfirm: () => void;
+  overrideTitle?: string;
+  overrideContent?: string;
+  overrideSubContent?: string;
   isCancel?: boolean;
-  isRequestingChange?: boolean;
 }
 
 export const SubmitModal: React.FC<SubmitModalProps> = ({
+  formTitle,
   handleClose,
   handleConfirm,
+  overrideTitle,
+  overrideContent,
+  overrideSubContent,
```

Comment:
We can't have the context `useMinisterHousingAllowance` on the submit form, as it will be used by SC, ASR, GC & MHA. We either make a shared context for all reports, or we pass in the `pageType`